include/swift/SIL/SILModule.h

```diff
@@ -232,13 +233,12 @@ class SILModule {
   // The list of SILProperties in the module.
   PropertyListType properties;
 
-  /// This is the underlying raw stream of OptRecordStream.
+  /// The remark file and the streamer used to serialize SIL remarks to a file.
   ///
-  /// It is also owned by SILModule in order to keep their lifetime in sync.
-  std::unique_ptr<llvm::raw_ostream> OptRecordRawStream;
-
-  /// If non-null, the YAML file where remarks should be recorded.
-  std::unique_ptr<llvm::yaml::Output> OptRecordStream;
+  /// The streamer is also owned by SILModule in order to keep their lifetime in
+  /// sync.
+  std::unique_ptr<llvm::raw_fd_ostream> SILRemarkStream;
```

Comment:
Since this is a doxygen comment, can you provide separate split docs for each of the ivars.